### ISSUE TITLE
Trying to use Anchor from react-buttons results in SVG parse error

### ISSUE THREAD

**Issue by rybesh** [zendeskgarden/react-components#281]
## Expectations

Trying to use the Anchor component from react-buttons:
```js
const { Anchor } = require( '@zendeskgarden/react-buttons')
```

## Reality

I get a parse error due to Anchor trying to `require` a raw SVG file:
```
ParseError: Unexpected token
line 1
place-picker/node_modules/@zendeskgarden/svg-icons/src/12/new-window-stroke.svg
```

Looking at the distributed file (`node_modules/@zendeskgarden/react-buttons/dist/index.js`) I see:
```js
module.exports = require("@zendeskgarden/svg-icons/src/12/new-window-stroke.svg");
```

My understanding is that the zendeskgarden build process is supposed to use SVGR to turn the raw SVG into React components, but somehow a `require` of raw SVG made it into the dist file?

Or is there some build step I'm missing here?  

### Fine Print

* Component: buttons
* Version: 3.9.0
* Browsers: all

**Comment by ryanseddon** [comment]
Hey @rybesh, thanks for the bug report. 

If you do not need the external icon for the `Anchor` component then an immediate fix to unblock you is to specify `@zendeskgarden/react-buttons@3.6.12` which is the latest version before we switched to `@svgr/webpack` to handle our svg stuff. 

We'll work on a fix to get this working and update you here when we've got something.

**Comment by austin94** [comment]
@rybesh this should now be corrected in `@zendeskgarden/react-buttons@3.9.1`

Thanks for helping us find this issue! Please re-open if you continue to see any issues.

**Comment by rybesh** [comment]
Thank you for the quick fix.
